static/gulpfile.js

```diff
@@ -0,0 +1,104 @@
+/* @file gulpfile.js
+ * @brief Builds front-end static files.
+ *
+ * @author Oscar Bezi (oscar@bezi.io)
+ */
+'use strict';
+
+//==============================================================================
+// Includes
+//==============================================================================
+var gulp = require('gulp');
+var util = require('gulp-util');
+
+var watchify = require('watchify');
+var browserify = require('browserify');
+
+var source = require('vinyl-source-stream');
+var buffer = require('vinyl-buffer');
+
+var assign = require('lodash.assign');
+
+var uglify = require('gulp-uglify');
+
+//==============================================================================
+// Config
+//==============================================================================
+var srcDir = './src/'
+var destDir = './build/'
+var bundleFiles = ['admin.js', 'index.js'];
+
+/* @brief Makes a JS build task for the given bundle files.
+ *
+ * We need a separate browserify bundling task for each destination bundle, so
+ * we pass it in to this function which then attaches the correct task to gulp
+ * under its name (for example, makeJSTask('index.js') would create a task
+ * 'index.js' that creates a destination bundle 'js/index.js' and starts
+ * watching for changes.
+ *
+ * @param {string} fname The filename of the entrypoint into the bundle,
+ * relative to `${srcDir}`/js/.
+ */
+var makeJSTask = function (fname) {
+  var handleErr = (err) => {
+    util.log(util.colors.red(`Browserify Error: ${err.message}`));
+  };
+
+  var customOpts = {
+    entries: [`${srcDir}/js/${ fname }`],
+    // set to true for sourcemapping
+    debug: false,
```

Comment:
In fact, I'd actually recommend you look into this. Right now, you don't have a way of building the JavaScript files non-interactively. That is, the only option to build JavaScript files is to run watchify. For deployment purposes, you're probably going to want to automate this.